girder/plugin.py

```diff
@@ -42,6 +42,7 @@ def registerPluginStaticContent(plugin: str, css: List[str], js: List[str], stat
             '/': {
                 'tools.staticdir.on': True,
                 'tools.staticdir.dir': str(staticDir),
+                'tools.staticdir.content_types': {'cjs': 'application/javascript'},
```

Comment:
Let's also add this to Girder core here: https://github.com/girder/girder/blob/v4-integration/girder/utility/server.py#L29-L42

If we add it using the `mimetypes` mechanism, I think it will automatically apply to all plugins too.

Reply:
That works too!